docs/controller/database_controller.md

```diff
@@ -0,0 +1,72 @@
+# Example controller with basic SELECT SQL Query
+
+1. Create `model/get_users_struct.go` structure in model folder
+
+```
+package model
+
+type User struct {
+	UserID string `json:"user_id"`
+	Name   string `json:"name"`
+}
+```
+
+2. Create API Controller in `controller/api/get_users.go`, which uses this struct to fill the data
```

Comment:
Dunno dunno about this. I think it should have been in the repository folder as its repository db function, no logic done. And some controller, service done to use that getusers repository function. But I am not sure. As you have only controller layer and you return model of users endpoint right away if that's supposed to be this way?

Reply:
I think the service and repository makes the code harder to understand, let it be right now i think.